Improve getting started (#7)
konstellation-io/website: content/en/docs/KRE/getting_started/index.md

weight: 5
---

## Installation

> The best way to start working with KRE, to test it and understand how it works, is deploy it on your local machine and follow a basic instructions to create an example project.

Reviewer: change:
-  is deploy for is **to** deploy and a basic instructions for basic instructions '

> The best way to start working with KRE, to test it and understand how it works, is deploy it on your local machine and follow a basic instructions to create an example project.

Reviewer 2: ```suggestion
The best way to start working with KRE, to test it and understand how it works, is deploying it on your local machine and follow some basic instructions to create an example project.
```

Once you are logged in as an administrator, you can create a new Runtime by simply following the [create Runtime guide]({{< relref "docs/KRE/tasks/create_runtime" >}}).

> ### 2. Download an sample KRT file

Reviewer 2: ```suggestion
### 2. Download a sample KRT file
```

> Inside a Runtime you can upload one or more versions that can later be ran and publish. You can use a pre-generated [sample greeter krt](/website/krts/greeter-v1.krt). You can find a more detailed explaination of how to create your own KRT file [here]({{< relref "docs/KRT/tasks" >}}).

Reviewer 2: ```suggestion
Inside a Runtime you can upload one or more versions that can later be ran and publish. You can use a pre-generated [sample greeter krt](/website/krts/greeter-v1.krt). You can find a more detailed explaination about how to create your own KRT file [here]({{< relref "docs/KRT/tasks" >}}).
```
